when-shit-hits-the-fan.adoc

```diff
@@ -381,29 +381,28 @@ everyone is downloading the whole block index at once.
 I'll build releases shortly.
 ____
 
-He wanted people to download block data from a specific user,
+He wanted people to download block data from a specific user, namely
 knightmb, who had published his blockchain as it appeared on his disk,
 the files blkXXXX.dat and blkindex.dat. The reason for downloading the
 blockchain data this way, as opposed to synchronizing from scratch, was
-said to reduce network bandwidth bottlenecks.
+to reduce network bandwidth bottlenecks.
 
-There was a big caveat with this: The data users would download from
-knightmb https://bitcoin.stackexchange.com/a/113682/69518[wasn't
+There was a big caveat with this: all the data users would download from
```

Comment:
I think you should revert this change back to just "the". I actually think none of the data was verified.

Reply:
Ok sorry, I misinterpreted